pom.xml

```diff
@@ -527,8 +527,8 @@
                     future, could consider switching to https://github.com/open-toast/gummy-bears
                     which accounts for Android desugaring and might allow usage of more Java classes -->
                   <groupId>net.sf.androidscents.signature</groupId>
-                  <artifactId>android-api-level-21</artifactId>
-                  <version>5.0.1_r2</version>
+                  <artifactId>android-api-level-26</artifactId>
```

Comment:
I saw that your commit description mentions that you were thinking about putting this in a separate PR.

Alternatively, you can introduce your own annotation for Animal Sniffer suppressions (likely copying the standard name, "`IgnoreJRERequirement`"), [configure your build to respect it](https://github.com/google/truth/blob/55d3fa76c482565afde5ea84d4021999705c19f2/pom.xml#L290), and then suppress at the narrow scope so that we continue to test that we're compatible with 21 (or 23, since that's the typical Google default nowadays) except in code that we're careful to run only under newer versions.

Reply:
I did end up doing that, thanks.